tests/test_integration/test_code_generation.py

```diff
@@ -182,7 +182,7 @@ def stencil(field_in: gtscript.Field[np.float_], field_out: gtscript.Field[np.fl
     np.testing.assert_allclose(field_out.view(np.ndarray)[:, :, -1], 2)
 
 
-@pytest.mark.parametrize("backend", OLD_BACKENDS)
+@pytest.mark.parametrize("backend", CPU_BACKENDS)
```

Comment:
It would be nice to have a test for the cases that are not allowed, e.g. writing to a 2D Field in a PARALLEL computation. Can we also test GPU if available without much effort here?

Reply:
I propose to not increase the CI burden for this, as the conceivably common cases are caught in the frontend and lowerings and are currently unit tested there.

Reply:
Ok, fine with me.